### ISSUE TITLE
All my work is gone

### ISSUE THREAD

**Issue by lalo2302** [HappenApps/Quiver#784]
Yesterday I was taking notes, closed my macbook (didn't turned it off) and today everything I wrote yesterday is gone. This is a deal breaker, I don't trust your product anymore

**Comment by gramster** [comment]
Happened to me too - all gone! Months of notes.

**Comment by jeffkimbrel** [comment]
This is a place to report bugs... not complain. There is literally no way anybody can help when you don't give any actually useful information to replicate or troubleshoot.

**Comment by gramster** [comment]
We opened the app today and our work was gone.  Of course that counts as a bug.

The files are present but Quiver does not see them. Only the tutorial.

**Comment by jeffkimbrel** [comment]
You could have said something like "I tried what was mentioned in issue #583 (or #686, #691, #701, #709, #733, etc.) but in my situation it didn't work because reason x, y and z."

You could have said "I can still navigate to it in Finder, yet the notes are not displaying in the application."

You could have said "Even after a restore from time machine, the data gets lost again after opening the application."

You don't have to call the people doing the homework for you an "idiot".

**Comment by gramster** [comment]
Note that if you click on 'Report Issues' in the app it takes you directly to a new ticket here in GitHub, so that is the mechanism the developer is expecting. Some person was complainiung earlier here about these being complaints, not bugs.

**Comment by gramster** [comment]
Trying to directly open Quiver.qvlibrary from the filesystem also shows nothing but the tutorial, despite the notes being present in JSON files.

**Comment by gramster** [comment]
@lalo2302, if you don't need mobile access, an alternative is Jupyter (or nteract if you want a native app). Quiver hasn't been updated in 8 months and the mobile app seems to have largely stalled too. The dev doesn't seem very responsive any more, and even his Twitter account has been silent for about 8 months. I suspect it may be a dead project at this point.

**Comment by kimaldis** [comment]
I'm inclined to agree; there's been very little response for a while now. It's a pity, it's so close to being so good.
I'm trialling Mweb just now. It seems to be working well for me so far.

**Comment by lalo2302** [comment]
So sad this project is abandoned. Would be awesome if the developers open source it. @gramster I tried Jupiter, but seems too much for what I need. A simple markdown native note editor with images and formatting is what I need, but apparently there is no alternative

**Comment by gramster** [comment]
Perhaps try Bear? It's very good.

**Comment by rockyzhang24** [comment]
@gramster Yep, Bear is good enough but I am not willing to give up Quiver. Hope the developer can shock us by releasing the new version in a few days. Lol.

**Comment by gramster** [comment]
Mweb looks interesting - quite similar to Quiver without the hassle of multiple cells, which never really made sense in Quiver without cell execution a la Jupyter. Would be interested in hearing your further experience.

**Comment by lalo2302** [comment]
Bear looks great, but it doesn't mention anywhere about notebooks, or some way to organize your notes beyond tags. The developer just email me and said the project was not abandoned, though he's been busy lately.

**Comment by gramster** [comment]
You can nest tags in Bear, and use that for organizing. It works pretty well. E.g. I have things like:

#1-1s/alec
#1-1s/john
#weeklies/20160110

etc, and it gives me  pseudo file tree.

**Comment by kimaldis** [comment]
Bear tags are very elegant. I particularly like that you can drop them anywhere in documents. They only nest two levels, though and that's only a small step up from Quiver. Although there is a little pressure to increase that. Developers seem quite active.

I'm liking Mweb very much right now. My only concern being it seems to be driven by a one man team; I'd be worried about support longevity.

**Comment by rockyzhang24** [comment]
The support of code in Quiver is stronger and powerful than MWeb, which is the most competitive part for Quiver and the only feature that keep me from giving up.

**Comment by ChristianClassen** [comment]
Just moved all my markdown notes from Quiver to [Boostnote](https://boostnote.io/), a free, "open source note taking app for programmers". Really like the experience so far.

**Comment by liweijian** [comment]
<img width="455" alt="05ce30c7-4072-4434-80f7-1c1b5afedf64" src="https://cloud.githubusercontent.com/assets/19320783/23126110/38f8c3de-f7b0-11e6-97b0-53652784a498.png">

@b2e9f523 Any tutorial on moving Quiver's notes to Boostnote?

**Comment by ChristianClassen** [comment]
@liweijian Unfortunately, I can't provide you with a tutorial. I had around 30 markdown-only (no other cells separating the content) notes so it was the easiest way for me to just copy-paste those into Boostnote.

But what you can do is:
1. Choose a note OR a complete notebook
2. Right click
3. Export as ... > Markdown
4. Creates a folder with each note in Markdown

**Comment by liweijian** [comment]
I have to use Time Machine to recover my system to 10.12.2, than exported all my 300+ notes.

**Comment by luyunfei** [comment]
why not use git to manager your version?

**Comment by medifle** [comment]
@b2e9f523 The search function of Boostnote is not that elegant

**Comment by kimaldis** [comment]
And it doesn't appear to support nested notebooks. It seems very bare bones to me.

**Comment by ChristianClassen** [comment]
@medifle @kimaldis Yes, both of you are absolutely right. It fits my use case and is under active development, that's why I switched to it. Please have a look at the [Roadmap](https://github.com/BoostIO/Boostnote/wiki/List-of-the-requested-features) and open a pull request for features you would like to see.

**Comment by jeffmax** [comment]
I've had the same issues described here (only able to open the tutorial). I was able to use the "restore from backup" feature to copy my old notebook back in. That being said, this really needs to be fixed.

**Comment by baitouwei** [comment]
Same here,lose all note

**Comment by baitouwei** [comment]
Luckily, find file in `/Users/***/Library/Mobile Documents/com~apple~CloudDocs/Quiver`

**Comment by ylian** [comment]
@baitouwei Can you check your Quiver library inside iCloud Drive? Is the library file completely gone? or renamed?

**Comment by baitouwei** [comment]
@ylian The library file is well in iCloud Drive. It seems the Quiver can read it from iCloud Drive.
My system version 10.12.2, Quiver version 3.0.4 (96).

![image](https://cloud.githubusercontent.com/assets/1253712/24319167/aad90326-114e-11e7-9565-f44ca642b197.png)
￼
![image](https://cloud.githubusercontent.com/assets/1253712/24319175/2c2fb7d0-114f-11e7-9d4f-502a8b9cf006.png)

**Comment by ylian** [comment]
@baitouwei If you just double click Quiver.qvlibrary and open it, do you see your notebooks and notes? If not, can you right click on Quiver.qvlibary, show package contents, and see if your notebooks are there? Thanks.

**Comment by baitouwei** [comment]
@ylian Doube click Quiver.qvlibrary and it loads notes success, find all my notes.

**Comment by ylian** [comment]
@baitouwei That's great! So the notes are intact in the library file. Could you tell me what you saw when you had the problem? An empty window? See only default notebooks such as tutorials? It will help me narrow down the issue. Thanks.

**Comment by baitouwei** [comment]
@ylian I use Quiver normal, the last day I write a note is 2017.3.18.But yesterday I open Quiver and it shows me an empty window without anything.So I want to sync it from iCound Drive, then I click `Open an Existing Library...` and find there is no way to select anything from iCould Drive.See the pic I send, it only show a `Remote Disc`

**Comment by ylian** [comment]
@baitouwei I see. Thanks a lot for the details. I didn't know iCloud Drive might do that. At the moment you could just double click the library to open if this happens again. I will try to reproduce with iCloud Drive.

**Comment by CalebSDHUB** [comment]
I got into the same issue. The initial project library disappeared (I never checked any files before it happened). So my guess is that it never actually created a quiver library. file, so I created a new quiver library and tried again. Now I am able to see the  .qvlibrary format at least.
